packages/server/src/stacks-controller.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from 'express';
+import cors from 'cors';
+import { makeSTXTokenTransfer, TransactionVersion } from '@blockstack/stacks-transactions';
+import BigNum from 'bn.js';
+
+export const StacksController = express.Router();
+
+StacksController.use(cors());
+
+StacksController.post('/faucet', (req: Request, res: Response) => {
```

Comment:
What do you mean by faucet here?

Thought I'd double check 
![image](https://user-images.githubusercontent.com/1618764/77761598-97792580-7038-11ea-99d2-5b51e658e127.png)

Reply:
"Faucet" in the crypto world refers to a way to get free crypto, typically on a testnet (although there were mainnet BTC faucets back in the day!).

https://en.wikipedia.org/wiki/Bitcoin_faucet